civihr_default_theme/scss/drupal/_form.scss

```diff
@@ -400,3 +400,51 @@
     width: 50%;
   }
 }
+
+.form-radios {
```

Comment:
> Radio buttons styles were previously declared for Onboarding Wizard only in _onboarding-wizard.scss, those has been moved to drupal/_form.scss.

why was this necessary?

Reply:
Previously when onboarding wizard was done, I did not ask the question that "whether the radio button styles should be made global" and went for the short route i.e. styling only for onboarding wiz, apologies. But now I have confirmed with Jon that the radio button styles are global.